### ISSUE TITLE
The final test accuracy

### ISSUE THREAD

**Issue by ustctf** [gpleiss/efficient_densenet_pytorch#34]
Hi,  Thanks for this implementation ! I'm wondering how to obtain the quite strong test set result on CIFAR-10, as reported in the original densenet paper (e.g., error rate <=3.5 on C-10+, with depth =190, growth_rate = 40). When I run the script as:

_CUDA_VISIBLE_DEVICES=0,1,2,3 python demo.py --depth 190 --efficient False --data ./data --save ./ckpts_

The final test error is reported as 0.0535. I'm wondering whether the high error is due to no data augmentation is conducted in the default setting. May I know whether it is C10+ dataset or C10?  

Best

**Comment by taineleau** [comment]
Could you please show the configuration (e.g., pytorch version, python version)?

**Comment by gpleiss** [comment]
I noticed recently that I was missing the appropriate initialization of the models. Try it again and let me know what the error is.

**Comment by gpleiss** [comment]
With the new initialization in c609c0c4 I was able to match the errors reported in the original paper. Closing this issue for now. If you're still noticing issues feel free to reopen.

**Comment by ustctf** [comment]
@gpleiss 

Sorry but I got the same test set error rate (>5) using your new initialization.

The command:

`CUDA_VISIBLE_DEVICES=0,1,2,3 python demo.py --depth 190 --efficient False --data ./data --save ./ckpts`

Can you show me your script to match the claimed error rate?

BTW the python version: Python 3.6.1 |Anaconda custom (64-bit)| 
pytorch version:  0.3.1.post2
gpu: 4 titanXp

Thanks.

**Comment by gpleiss** [comment]
python demo.py --depth 100 --efficient False --data ./data --save ./ckpts --batch_size 64 --valid_size 0

These are the same hyperparameters as in the paper. I got a final test error of 0.047

**Comment by ustctf** [comment]
Thanks @gpleiss , I can reproduce now and with --depth 190, --growth_rate 40, the test error is 0.035.
